db/sg_replicate_cfg.go

```diff
@@ -492,7 +492,11 @@ func (m *sgReplicateManager) RefreshReplicationCfg() error {
 			if err != nil {
 				base.Warnf("Unable to gracefully close active replication: %v", err)
 			}
+			if !ok {
+				activeReplicator.purgeStatus()
```

Comment:
Given purgeStatus returns `nil` for not found errors, I think we could probably log in this case? Otherwise explicitly ignore to avoid errcheck complaining.

```suggestion
				if err := activeReplicator.purgeStatus(); err != nil {
					base.Warnf("Unable to purge status of deleted replication %s: %v", replicationID, err)
				}
```